sermant-agentcore/sermant-agentcore-core/src/main/java/io/sermant/core/service/xds/XdsFlowControlService.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (C) 2024-2024 Sermant Authors. All rights reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package io.sermant.core.service.xds;
+
+import io.sermant.core.service.xds.entity.XdsCircuitBreakers;
+import io.sermant.core.service.xds.entity.XdsHttpFault;
+import io.sermant.core.service.xds.entity.XdsOutlierDetection;
+import io.sermant.core.service.xds.entity.XdsRateLimit;
+import io.sermant.core.service.xds.entity.XdsRetryPolicy;
+
+import java.util.Optional;
+
+/**
+ * xDS FlowControl service
+ *
+ * @author zhp
+ * @since 2024-11-27
+ **/
+public interface XdsFlowControlService {
+    /**
+     * get circuit breaker information of cluster
+     *
+     * @param serviceName service name
+     * @param clusterName cluster name
+     * @return circuit breaker rules
+     */
+    Optional<XdsCircuitBreakers> getCircuitBreakers(String serviceName, String clusterName);
+
+    /**
+     * get Outlier Detection of cluster
+     *
+     * @param serviceName service name
+     * @param clusterName cluster name
+     * @return Outlier Detection rules
+     */
+    Optional<XdsOutlierDetection> getOutlierDetection(String serviceName, String clusterName);
```

Comment:
CircuitBreaker should contain OutlierDetection

Reply:
done